Makefile

```diff
@@ -14,22 +14,8 @@ test: all
 test-cov: all
 	py.test --cov=q2_feature_table
 
-q2_feature_table/_summarize/summarize_assets/dist:
-	cd q2_feature_table/_summarize/summarize_assets && \
-	npm install && \
-	npm run build && \
-	cp licenses/* dist/
-
-viz-summarize: q2_feature_table/_summarize/summarize_assets/dist
-
 install: all
 	$(PYTHON) setup.py install
 
 dev: all
 	pip install -e .
-
-clean: distclean
```

Comment:
Can you please put `clean` and `distclean` back in? They are part of the public API of the Makefile. A good example of a no-op variant can be found here: https://github.com/qiime2/q2-alignment/blob/master/Makefile#L23-L25